text/3724-theupdateframework.md

```diff
@@ -0,0 +1,341 @@
+
+Rustaceans need to be able to download crates and know that they're getting the crate files that were published to crates.io without modification.
+
+There are places where Rust is difficult to use right now. Using Cargo with crates.io works well for Rustaceans with unfirewalled access to high speed Internet, but not all are so lucky. Some are behind restrictive firewalls which they are required to use. Some don't have reliable access to the Internet. In cases like these, we want to support mirrors of crates.io in a secure way that provides cryptographic guarantees that they are getting the same packages as are provided by the Rust Project, without any risk of tampering.
+
+Another reason for wanting to be able to better support mirrors is to address cost pressures on Rust. Approximately half of Rust release and crate traffic is from CI providers. Being able to securely distribute Rust crates from within CI infrastructure would be mutually beneficial, since it would both allow the Rust Foundation to reallocate budget to other uses and would make Rust CI actions faster and more reliable on those platforms.
+
+Finally, supply chain security is a growing concern, particularly among corporate and government users of Rust. The Log4j vulnerability brought much greater attention to the problems that can occur when a single dependency nested arbitrarily deep in a dependency graph has a critical vulnerability. Many of these users are putting significant resources into better understanding their dependencies, which includes being able to attest that their dependencies verifiably came from specific sources like crates.io.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+For an understanding of terms utilized in this section, please see the [TUF reference](https://theupdateframework.github.io/specification/latest/).
+
+#### What is TUF?
+
+[The Update Framework](https://theupdateframework.io/) (TUF) is a software framework designed to protect software update repositories that automatically identify and download updates to software. TUF uses a series of roles and keys to provide a means to retain security, even when some keys or servers are compromised. It does this with a stated goal of requiring minimal changes and effort from repository administrators, software developers, and end users. ([Wikipedia](https://en.wikipedia.org/wiki/The_Update_Framework))
+
+TUF provides a methodology and framework for managing and verifying a modern chain of trust between what it terms "roles", stemming from a root and allowing for a further delegation of roles for different functions and paths within the repository.
+
+At the simplest level, TUF is a collection of signing key quorums which verify its own integrity, and establishes trust between the quorums and files within a repository.
+
+#### Terminology
+
+- `tuf`: The Update Framework and its specification
+- `targets`: The actual content and files distributed and to be signed
```

Comment:
`target` is already an overloaded term in the Rust/Cargo world, we may want to be careful how we use it in documentation of this RFC to avoid having an additional meeting we need to disambiguate.